glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -22,10 +22,11 @@ class GleanTest {
     @Test
     fun `send a ping`() {
         val context: Context = ApplicationProvider.getApplicationContext()
-        val glean = GleanInternalAPI()
-        glean.initialize(context)
-        glean.handleBackgroundEvent()
+        Glean.testDestroyGleanHandle()
```

Comment:
Instead of doing this and the line below, let's just call the shared `resetGlean`. You can do so by adding this block of code at the beginning of the class, before any test definition:

```
    @Before
    fun setUp() {
        resetGlean()
    }
```

Note: I don't think we need the simple one line "smoke test" anymore